### PR TITLE
Fix 1917 - Autocorrect wilcard rules on never cache urls

### PR DESCRIPTION
**Pull request by crystinutzaa**
Autocorrect wildcard rules on Never Cache Url(s)

Incorrect wildcard rules as * , .* will be automatically replaced by the correct rule (.*)

Based on:
3.4.1 WP Rocket Minor Version doc

### CODE REVIEW

File: inc/functions/formatting.php [hunk 1]
```diff
@@ -20,6 +20,35 @@ function rocket_clean_exclude_file( $file ) {
...
+ */
+function rocket_clean_wildcards ( $path ) {
+	if ( ! $path ) {
+		return false;
+	}
+
+	$path_components = explode( '/', $path );
+    $arr             = array(
+            ".*"   => "(.*)" ,
+            "*"    => "(.*)" ,
```
Comment (Screenfeed) [inline inc/functions/formatting.php]: I suppose we could also add `'(*)' => '(.*)',`.
Also, phpcs won't be happy.

Comment (Screenfeed) [reply inc/functions/formatting.php]: Nice use of `strtr()` 👍

Comment (crystinutzaa) [reply inc/functions/formatting.php]: Added also: `'(*)' => '(.*)' rule`

Applied to CSS/JS files exclusions, User Agents exclusion, Always Purged URLs, and CDN exclusions

Comment (Screenfeed) [reply inc/functions/formatting.php]: Last changes about styling:
- `$arr             = array(`: short array syntax please.
- `".*"   => "(.*)" ,`: no space character before the last `,` for the 4 lines.

Comment (crystinutzaa) [reply inc/functions/formatting.php]: Coding style changes applied. Also I changed all array() into [] shorthand. It's confusing to see array()

**Pull request merged**